2022/2022-03-22.md

```diff
@@ -20,7 +20,7 @@ Start: 2:00 pm UK
   - Ongoing search
 
 - NGFF:
-  - See [notes](/0aD4FfzCS5C5TWbLmk-cxA) from the extended discussion on rich metadata in OME-Zarr
+  - See [notes] from the extended discussion on rich metadata in OME-Zarr
```

Comment:
```suggestion
  - See [notes](https://hackmd.io/0aD4FfzCS5C5TWbLmk-cxA) from the extended discussion on rich metadata in OME-Zarr
```

The absolute `/...` here only works on hackmd itself. When moving to www.o.org it needs the absolute URL